src/pages/transaction-signing/components/event-card.tsx

```diff
@@ -21,7 +22,7 @@ export const TransactionEventCard: React.FC<any> = ({
       <Stack spacing="base-loose" p="base-loose">
         <SpaceBetween position="relative">
           <Text fontWeight={500} fontSize={2}>
-            {title}
+            <span data-testid={SendFormSelectors.TransferMessage}>{title}</span>
```

Comment:
Apply id to `<Text>` element, a new span isn't necessary

Reply:
I have problems grabbing the element with just the <Text> I do this to avoid flakes. It works but the test is brittle and breaks for me on some runs.

Reply:
I have a feel that's nothing to do with the selector being on the text element (which underneath is a span). It's either there or it isn't.

Reply:
this rebase is not working for me

Reply:
what's going wrong?

Reply:
Let jump on a zoom when you have a free second the rebase isnt on a branch

 (no branch, rebasing chore/regressionSep27)

Reply:
resolved